Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -123,7 +123,7 @@ private function grabAttribsBeforeMethodToken(Tokens $tokens, $index)
      * @param Tokens $tokens Tokens collection
      * @param int    $index  token index
      *
-     * @return array array of grabbed attributes
+     * @return array<string, Token|null> array of grabbed attributes
```

Comment:
It seems it really has been a while since I read the spec. Seems that change was reverted on 8th August 2014, lol. https://github.com/phpDocumentor/fig-standards/commit/61c695c43d1b2c713acbfb3302f284c518421bc5